examples/enclosures.py

```diff
@@ -128,6 +128,52 @@
 except HPOneViewException as e:
     print(e.msg)
 
+# Create a Certificate Signing Request (CSR) for the enclosure.
+bay_number = 1  # Required for C7000 enclosure
+csr_data = {
+    "type": "CertificateDtoV2",
+    "organization": "",
+    "organizationalUnit": "",
+    "locality": "",
+    "state": "",
+    "country": "",
+    "commonName": ""
+}
+try:
+    oneview_client.enclosures.generate_csr(csr_data, enclosure_uri, bay_number=bay_number)
+    print("Generated CSR for the enclosure.")
+except HPOneViewException as e:
+    print(e.msg)
+
+# Get the Certificate Signing Request (CSR) that was generated by previous POST.
+try:
+    csr = oneview_client.enclosures.get_csr(enclosure_uri, bay_number=bay_number)
+    with open('enclosure.csr', 'w') as csr_file:
+            csr_file.write(csr["base64Data"])
+    print("Saved CSR(generated by previous POST) to 'enclosure.csr' file")
+except HPOneViewException as e:
+    print(e.msg)
+
+# Import signed ecrtificate to the enclosure.
+try:
+    # Create a CA-signed certificate using the CSR saved to "enclosure.csr" file.
+    user_input = raw_input("Save the signed certificate to certificate.cer and press 'y' to continue with certificate import or press 'n'")
```

Comment:
My suggestion:
To me, It makes more sense to allow the user to specify the name of the 'certificate.ce' file at the beginning of this example file, since that file could be named anything.
There we could have a comment explaining that this action requires the certificate name to be declared, and here, instead of using `raw_input`, we could have a simple `if` checking if the user specified the `certificate_file`.

Reply:
Agree with @fgbulsoni on this.

Reply:
Intention was to let user know that he has to use the CSR generated from the previous request but you are correct as an example file no need to interrupt the script execution.

Updated code.